SPECS/mysql/mysql.spec

```diff
@@ -25,7 +26,7 @@ Requires:       %{name} = %{version}-%{release}
 Development headers for developing applications linking to maridb
 
 %prep
-%setup -q %{name}-boost-%{version}
+%autosetup -p1
```

Comment:
Are you sure the package builds now with `%autosetup`? I recall having issues using that, if the spec had any .nopatch "patches".

Reply:
For what it's worth, I only have this problem when there's text in the nopatch files.

Reply:
I successfully built the mysql package locally using autosetup. The only issue I had was since '%{name}-boost-%{version}'  wasn't being called with -n, it was effectively doing nothing in the %setup. Thus, I removed it (thanks for help with that Henry :) ). Besides that, I had no problems with %autosetup